utils/searx.sh

```diff
@@ -88,7 +95,8 @@ BUILD_PACKAGES_centos="\
 firefox graphviz graphviz-gd ImageMagick librsvg2-tools
 texlive-xetex-bin texlive-collection-fontsrecommended
 texlive-collection-latex dejavu-sans-fonts dejavu-serif-fonts
-dejavu-sans-mono-fonts fontforge ttfautohint"
+dejavu-sans-mono-fonts fontforge ttfautohint
+"
```

Comment:
No npm for this one ?

Reply:
These are the yum packages for cenos-7 (we have support only for centos-7). Modern versions of centos, fedora and RHEL are using dnf package manager. Automation of the yum package installation is more complicate and centos-7 was old from the beginning when we added it.  I vote to drop these old distribution.

Comiting the newline was a fail of mine / I will remove the newline.